pymc/distributions/__init__.py

```diff
@@ -191,6 +192,7 @@
     "CAR",
     "PolyaGamma",
     "joint_logpt",
```

Comment:
Remove `joint_logpt` from `__all__`

Reply:
Wouldn't this break people's code instead of issuing the FutureWarning?

Reply:
`__all__` is just relevant for `from xyz import *` imports.
But we're using start imports in `__init__.py`, so you're right that it would break user code.
So don't remove it yet.